src/JSEncrypt.ts

```diff
@@ -14,21 +15,21 @@ export interface IJSEncryptOptions {
  *
  * @param {Object} [options = {}] - An object to customize JSEncrypt behaviour
  * possible parameters are:
+ * - key                     {JSEncryptRSAKey}  default: null
  * - default_key_size        {number}  default: 1024 the key size in bit
  * - default_public_exponent {string}  default: '010001' the hexadecimal representation of the public exponent
  * - log                     {boolean} default: false whether log warn/error or not
  * @constructor
  */
 export class JSEncrypt {
     constructor(options: IJSEncryptOptions = {}) {
-        options = options || {};
```

Comment:
I removed this as it was redundant due to the utilization of the default parameter.